api/users/views.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import views
+from rest_framework import viewsets
+from rest_framework.response import Response
+
+from api.users.serializers import ShareUserSerializer
+from api.users.serializers import ShareUserWithTokenSerializer
+
+from share import models
+
+
+class ShareUserViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Returns details about the currently logged in user
+    """
+    serializer_class = ShareUserSerializer
+
+    def get_queryset(self):
+        return models.ShareUser.objects.filter(pk=self.request.user.pk)
+
+
+# TODO Remove me
```

Comment:
This is probably related to moving things to `users/me`. We should update this comment and reference https://openscience.atlassian.net/browse/SHARE-586.